Spring-Project/src/main/java/com/epam/project/spring/controller/UserController.java

```diff
@@ -0,0 +1,49 @@
+package com.epam.project.spring.controller;
+
+import com.epam.project.spring.dto.UserDto;
+import com.epam.project.spring.model.User;
+import com.epam.project.spring.service.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
```

Comment:
Hi Zenyk. Please remove "/user" path from all ***Mapping** annotations on the method level and add `@RequestMapping("/user")` on the Controller level. Thank you

Reply:
Thanks. Resolved